src/Cli/dotnet/CommandFactory/CommandResolution/LocalToolsCommandResolver.cs

```diff
@@ -73,13 +74,20 @@ public CommandSpec Resolve(CommandResolverArguments arguments)
         }
 
         private CommandSpec GetPackageCommandSpecUsingMuxer(CommandResolverArguments arguments,
-            ToolCommandName toolCommandName)
+            ToolCommandName toolCommandName, bool allowRollForward = false)
         {
             if (!_toolManifest.TryFind(toolCommandName, out var toolManifestPackage))
```

Comment:
Tests should be added if allowRollForward is false in `LocalToolsCommandResolver`